src/serverModel.ts

```diff
@@ -190,12 +350,14 @@ export default function ServerModel(mpInstance) {
                 };
             }
 
+            // TODO: Should we move this side effect outside of this method?
             if (event.messageType !== Types.MessageType.SessionEnd) {
                 mpInstance._Store.dateLastEventSent = new Date();
             }
 
             uploadObject = {
-                Store: mpInstance._Store.serverSettings,
+                // TODO: Why are we passing Server Settings as Store?
+                Store: (mpInstance._Store.serverSettings as unknown) as IStore,
```

Comment:
This is definitely misleading.  The  `Store`  on the V2 SDK is something the server was expecting, and it is NOT the `IStore` which contains all the configuration and other SDK attributes.

Reply:
This is removed in V3.  I never knew what the server did with this.

Reply:
Note to self: Make this an any or empty object for now